unit_tests/web_tests/test_web_image.py

```diff
@@ -51,6 +52,72 @@ def test_web_image_upload_url(self):
         self.assertTrue(self.page.image_exists(image_name))
         wta.assertExists('image', image_name, group=self.gvar['base_group'], image_cloud=cloud_name)
 
+    # test the checkboxes-------------------------------------------------
+    def test_two_checkboxes(self):
+        self.page.click_upload_image()
+        virt_sparsify_checked = self.page.is_checkbox_selected("operation1")
+        with_compression_checked = self.page.is_checkbox_selected("operation2")
+
+        self.assertFalse(virt_sparsify_checked)
+        self.assertFalse(with_compression_checked)
+
+    def test_web_image_upload_filename_with_sparsify_compression_checkbox(self):
+        # Uploads an image to a cloud using a system file
+        image_name = self.gvar['user'] + '-wii4.hdd'
+        cloud_name = self.gvar['user'] + '-wic2'
+
+        # uploading .hdd file by clicking two checkboxes
+        self.page.click_upload_image()
+        self.page.type_image_file_path(helpers.misc_file_full_path(image_name))
+        self.page.click_checkbox("operation1")
+        self.page.click_checkbox("operation2")
+        self.page.add_upload_to_cloud(cloud_name)
+        with wti.wait_for_page_load(self.driver, timeout=1000):
+            self.page.click_upload()
+        self.page.click_top_nav('Images')
+
+        self.assertTrue(self.page.image_exists(image_name))
+        wta.assertExists('image', image_name, group=self.gvar['base_group'], image_cloud=cloud_name)
+        print("\ntester-wii4.hdd file successfully uploaded")
+
+        # delete web image upload file_name
+        self.page.click_top_nav('Images')
+        image_name = self.gvar['user'] + '-wii4.hdd'
+        cloud_name = self.gvar['user'] + '-wic2'
+        with wti.wait_for_page_load(self.driver, timeout=1000):
+            self.page.click_cloud_button(image_name, cloud_name)
+            self.page.click_delete_ok()
+        print("tester-wii4.hdd file successfully deleted")
+
+    def test_web_image_upload_url_two_checkboxes(self):
+        # Uploads an image to a cloud using a URL
+        cloud_name = self.gvar['user'] + '-wic2'
+        self.page.click_upload_image()
+        self.page.click_from_url()
+        self.page.type_image_url('http://cernvm.cern.ch/releases/production/cernvm4-micro-2020.07-1.hdd')
+
+        self.page.click_checkbox("operation1")
+        self.page.click_checkbox("operation2")
+
+        self.page.add_upload_to_cloud(cloud_name)
+        with wti.wait_for_page_load(self.driver, timeout=1000):
+            self.page.click_upload()
+        self.page.click_top_nav('Images')
+
+        self.assertTrue(self.page.image_exists("cernvm4-micro-2020.07-1.hdd"))
```

Comment:
Again we may want to use configuration for this value instead of static text since we are not hosting the file ourselves.